test/fhirdefs/load.test.ts

```diff
@@ -148,7 +148,7 @@ describe('#loadDependency()', () => {
       }
     ]);
     expect(requestSpy.mock.calls[1][0].uri).toBe(
-      'http://build.fhir.org/ig/nrdavis1/HSPCFHIRtest/branches/package.tgz'
+      'http://build.fhir.org/ig/nrdavis1/HSPCFHIRtest/package.tgz'
```

Comment:
I feel like this no longer totally tests that the sorting by date is working anymore, since no matter which package you get, the `repo` starts with `nrdavis1/HSPCFHIRtest`. I think the date sort would only matter if there was a package that was being published with the same `package-id` from two different repos. Which I don't see an example of in qas.json. But if we are going to leave the date sort in, I feel like we should still test it here by changing the repo of the older qa entry.

Reply:
Yeah, I mentioned the same thing to Grahame -- the date sort seems unnecessary at this point.  But in the off-chance that a package does switch repo names, I guess it might matter.

I take your point though.  If date sort was broken, we'd have no idea.  So I'll update the test.  I also realized this morning that I think my logic has an extra `slice(0,2)` that's totally not necessary since I'm using destructuring assignment (the slice is left over from an earlier version that did not use destructuring assignment).

Reply:
OK, @ngfreiter -- this is fixed in the last commit.  I also switched up the test a little so that poor unsuspecting nrdavis1 is no longer in our testcases.